libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -16,7 +16,7 @@ export const NavbarWrapper = styled('nav')`
 const appBarStyles = (theme: Theme, isMenuOpen: boolean) =>
   isMenuOpen
     ? css`
-        background-color: ${theme.palette.primary.main};
+        background-color: ${theme.palette.background.paper};
```

Comment:
it probably should stay `primary.main` as it's our new main color.
Paper background also effects a lot of other stuff we don't want and doesn't affect the buttons which we want afaik.